tools/debian/control

```diff
@@ -70,26 +74,26 @@ Description: Cockpit deployment and developer guide
  or extend Cockpit.
 
 Package: cockpit-docker
-Architecture: amd64 armel armhf i386
+Architecture: all
 Depends: ${misc:Depends},
          docker.io (>= 1.3.0) | docker-engine (>= 1.3.0),
-         python
+         cockpit-bridge (>= ${source:Version}),
+         cockpit-bridge (<< ${source:Version}.1~)
 Description: Cockpit user interface for Docker containers
  The Cockpit components for interacting with Docker and user interface.
 
 Package: cockpit-machines
-Architecture: any
+Architecture: all
 Depends: ${misc:Depends},
          libvirt-daemon-system
 Description: Cockpit user interface for virtual machines
  The Cockpit components for managing virtual machines.
 
 Package: cockpit-networkmanager
-Architecture: any
-# Lock bridge dependency due to --with-networkmanager-needs-root
-# which uses new less stable /manifests.js request path.
-depends: ${misc:Depends},
-         cockpit-bridge (= ${binary:Version}),
+Architecture: all
+Depends: ${misc:Depends},
+         cockpit-bridge (>= ${source:Version}),
+         cockpit-bridge (<< ${source:Version}.1~),
```

Comment:
It's fine for Debian/Ubuntu packages to keep this simple for now. 

When the cockpit meta package is installed it enforces that they all remain in sync. But going forward there's definitely the ability for various packages like cockpit-networkmanager or cockpit-storaged to be upgraded separately and have their version numbers out of sync when installed individually. There's a certain base minimum version number of the bridge that's required. You can see this in the RPM spec file.